### PR TITLE
ECO-688: Fixed validation message for incomplete credit card form.

### PR DESCRIPTION
**Pull request by ievgenvarava**
- Developer(s): @ievgenvarava

- Peer Reviewer: -

- Ticket: https://spryker.atlassian.net/browse/ECO-688

- Project PR: -

- Academy PR: - (for feature documentation and if needed migration guides and project integration guide)

#### Please confirm

- [x] No new OS components - or they have been approved by legal department
- [x] All new or heavily changed classes get an "A" rating (Scrutinizer), except Facades, Factories, QueryContainers and Tests

#### Documentation

- [x] Functional documentation provided or in progress?
- [x] Integration guide for projects provided or not needed?
- [x] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraints         |
   :--------------------- | :------------------------ | :--------------------- |
   Payone                | patch                 |                       |

#### Release Notes

Bugfixes:

* Fixed validation message for incomplete credit card form.

-----------------------------------------

#### Module Payone

_**Patch:** Backwards-compatible bug fix_

Def of done (by responsible developer):
- [x] All changes are backward-compatible. Outdated code is marked as deprecated
- [x] The change is isolated and not mixed with any cleanups or other changes
- [x] New code fits to the architecture rules
- [x] All new or changed business logic is covered by functional tests (in Zed business layer)

##### Change log

* Modified check() method to display default error message if credit card data is incomplete.
* Removed unused values from config.
* Added selector for error message container into config.

### CODE REVIEW

**Comment by dereuromark** [comment]
You can pickyback https://github.com/spryker-eco/payone/pull/5/files#r131967082

File: assets/Yves/js/modules/payment-method.js [hunk 1]
```diff
@@ -68,10 +68,11 @@ function check(iframes) {
     // Set it explicitly.
     window['checkCallback'] = checkCallback;
 
-    if (iframes.isComplete()) {
+    if (iframes.isComplete() && $(config.cardholderInput).val()) {
         iframes.creditCardCheck('checkCallback');
     } else {
         console.debug("The form is not complete");
```
Comment (dereuromark) [inline assets/Yves/js/modules/payment-method.js]: .debug should be removed from here, right?

Comment (ievgenvarava) [inline assets/Yves/js/modules/payment-method.js]: done.

**Comment by dereuromark** [comment]
Looks good.

**Pull request merged**